setup.cfg

```diff
@@ -10,4 +10,5 @@ SPLIT_PENALTY_AFTER_OPENING_BRACKET=0
 [nosetests]
 verbosity=2
 where=test
+#where=test/v1/test_bucket.py:TestListUnfinished
```

Comment:
Should this be here?

Reply:
yes, otherwise when I want to run a single test, I forget how to do it and spend time reinventing it over and over ;)

Reply:
Cool.